ldregistry/templates/main/_item-render.vm

```diff
@@ -19,15 +19,15 @@
   <div class="col-md-10">
     #parse( "main/_breadcrumbs.vm" )
   </div>
-  <div class="col-md-offset-1 col-md-1">
-    #showstatus( $item.getPropertyValue("reg:status") )
-  </div>
 </div>
 
 <div class="row">
   <div id="description" class="col-md-10">
     <h2>#if($register)$msg.get('register.heading', $entity.name) #{else}$msg.get('registerItem.heading', $entity.name) #end #if($versionNumber)- $msg.get('registerItem.version.heading', $versionNumber)#end</h2>
-    <p>  <strong>URI:</strong> <a href="#linkhref($entity)" title="$entity.uRI">$entity.uRI</a> </p>
+    <p>  <strong>URI:</strong> <span title="$entity.uRI" id="forItemURI">$entity.uRI</span> &nbsp;&nbsp;<a href="#" onkeydown="copyToClipboardKeydown(event, 'forItemURI', 'copyButtonID_forItemURI')" onfocus="copyToClipboardFocus('copyButtonID_forItemURI')" onblur="copyToClipboardBlur('copyButtonID_forItemURI')"><span title="Copy URI to clipboard" id="copyButtonID_forItemURI" onclick="copyToClipboard('forItemURI', 'copyButtonID_forItemURI')">&nbsp;</span></a></p>
+    <div class="col-md-offset-0 col-md-1">
+      #showstatus( $item.getPropertyValue("reg:status") )
```

Comment:
When the value is "experimental" it overlaps the start of the decription text that follows it.

Reply:
Looks good now.